airflow.yaml

```diff
@@ -66,6 +66,13 @@ pipeline:
       python -m build --wheel
       pip install dist/*.whl
 
+  - runs: # CVE-2024-6345 GHSA-cx63-2mw6-8hw5
+    # setuptools comes from airflow/providers/google/provider.yaml having
+    # gcloud-aio-auth>=4.0.0,<5.0.0 . gcloud-aio-auth 4 is backlevel and has
+    # setuptools in it's pyproject.toml 'tool.poetry.dependencies'
+    # The tldr; For that case it is not needed in runtime.
+    pip uninstall --yes setuptools
```

Comment:
I'm interested in input.  Both specific and generic. airflow's dependency tree grabs setuptools at 66.1.1. 

I'm 95+% sure that it doesn't get used in runtime.  It seems the right thing to do is just remove it. 

The dep tree for that is:

* airflow/providers/google/provider.yaml having -> gcloud-aio-auth>=4.0.0,<5.0.0
* gcloud-aio-auth/auth/pyproject.toml having tool.poetry.dependencies with setuptools = ">= 66.0.0, < 67.0.0"

The options then address the CVE then are
1. remove setuptools
2. upgrade setuptools

I'm interested  in thoughts on this.